src/compiler/scala/tools/nsc/tasty/bridge/ContextOps.scala

```diff
@@ -0,0 +1,307 @@
+package scala.tools.nsc.tasty.bridge
+
+import scala.annotation.tailrec
+
+import scala.reflect.io.AbstractFile
+import scala.tools.nsc.tasty.TastyUniverse
+import scala.tools.nsc.tasty.TastyName
+import scala.tools.nsc.tasty.TastyModes._
+
+trait ContextOps { self: TastyUniverse =>
```

Comment:
once TreeUnpickler can only see abstract types, then all the `FooOps` traits will still see the concrete types of symbolTable